.travis.yml

```diff
@@ -15,6 +11,14 @@ env:
     - ROS_BUILDFARM_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
 matrix:
   include:
+    - language: python
+      python: "3.6"
+      sudo: false
+      install:
+        - pip install catkin-pkg EmPy PyYAML rosdistro vcstool
+        - pip install coverage "flake8<3.7" flake8-docstrings flake8-import-order nose pep8 pyflakes
+      script:
+        - python setup.py nosetests -s --tests test
```

Comment:
I would lean towards using `pytest` nowadays. But that could also be done in a separate PR.

Reply:
Sure